src/main/java/br/com/sunty/models/course/dto/AdminNewCourseForm.java

```diff
@@ -0,0 +1,99 @@
+package br.com.sunty.models.course.dto;
+
+import br.com.sunty.models.category.SubCategory;
+import br.com.sunty.models.course.Course;
+import br.com.sunty.models.course.CourseVisibility;
+import br.com.sunty.models.instructor.Instructor;
+import br.com.sunty.repository.instructor.InstructorRepository;
+import br.com.sunty.repository.subcategory.SubCategoryRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.validation.constraints.*;
+
+public class AdminNewCourseForm {
+
+    @NotBlank(message = "{course.name.not.null}")
+    @Size(max = 255, message = "{course.name.size.max}")
+    private final String name;
+    @NotBlank(message = "{course.url.not.null}")
+    @Size(max = 255, message = "{course.url.size.max}")
+    @Pattern(regexp = "[a-z]+([a-z-]*)[a-z]", message = "{course.url.regex}")
+    private final String urlCode;
+    @Min(value = 1)
+    @Max(value = 20)
+    private final int timeToFinishInHours;
```

Comment:
Aqui seria legal uns ENTER pra separar visualmente os trechos de código!

```suggestion
    @NotBlank(message = "{course.name.not.null}")
    @Size(max = 255, message = "{course.name.size.max}")
    private final String name;
  
    @NotBlank(message = "{course.url.not.null}")
    @Size(max = 255, message = "{course.url.size.max}")
    @Pattern(regexp = "[a-z]+([a-z-]*)[a-z]", message = "{course.url.regex}")
    private final String urlCode;
  
    @Min(value = 1)
    @Max(value = 20)
    private final int timeToFinishInHours;
```